### ISSUE TITLE
profile page not displaying 

### ISSUE THREAD

**Issue by zleap** [friendica/friendica#5174]
### Friendica version you encountered the problem

see `example.com/friendica` on your Friendica node for the version information.

**Comment by zleap** [comment]
1.,  Not sure iif this is the correct place to report this.

1,   I am using social.isurf.ca
2. My profile page (in firefox) displays as a blank page (  https://social.isurf.ca/profile/zleap )
2a In chromium  it gives error
This page isn’t working
social.isurf.ca is currently unable to handle this request.
HTTP ERROR 500

I have tried to report this on the server support page at 
https://social.isurf.ca/network?cid=7948&f=&conv=1&nets=all&

I have not had a response to this as yet ,  however (to be fair) if there is a problem with the server then it is possible people can't see the message or I can't see the replies.

I only signed up to the service just over a week ago,  this problem seems to have started on June 1st,  which  just so happens to coincide with the release of 2018-05  ( https://friendi.ca/blog/ )  

Just wondered if there was in fact a known problem,  seems to be something may have happened as it is not just my profile that appears like that,  the rest of the site works fine,  I don't seem to be able to view profiles on the social.isurf.ca server,  

As I don't have that many contacts yet,   I can view a profile on another friendi.ca instance.

hope someone can help or help the admins of social.isurf.ca out,  Sorry again if this is the wrong place to post this. 
 
Paul

**Comment by tobiasd** [comment]
> Just wondered if there was in fact a known problem, seems to be something may have happened as it is not just my profile that appears like that.

No, it is not a known problem. Thanks for the report! Hopefully the admin of social.isurf.ca is aware of it. I think @realkinetix is the admin of the node, so maybe he sees this report.

**Comment by zleap** [comment]
Thanks,  and no problem,    yeah hopefully it is easy to fix and track down what has caused it. 

regards

Paul

**Comment by realkinetix** [comment]
Thanks for the report, I'll look in to it.

**Comment by realkinetix** [comment]
Hmm... I'd manually patched DFRN.php during the -rc process and now git is refusing to keep up to date with the current version.  I've had to download it manually from github but I'm concerned I'm going to have issues with it until it sorts itself out.

However, now that I've replaced it, it appears I've got a big backlog of stuff that's starting to come in.

**Comment by MrPetovan** [comment]
How did you patch it?

**Comment by realkinetix** [comment]
Literally by hand with vi.  I thought I would have git complain to me when I did a pull or checkout, but I don't recall it doing so when I switched back to master.  However, DFRN.php then had << merge bits in it and was causing the 500's.

After doing a git reset on the file and checking it out, I keep getting a 98065-byte DFRN.php instead of the current 100754-byte DFRN.php.  

So, I've done a backup, and used some suggestions I've found for doing a local branch save, then a git fetch and a git reset --hard origin/master

Things seem to be OK now, at least the DFRN.php is the right version.  Will be watching for errors for a while.
